docs/source/integrations/ultralytics.rst

```diff
@@ -230,6 +220,77 @@ manually convert Ultralytics predictions to :ref:`FiftyOne format <keypoints>`:
    :alt: ultralytics-keypoints
    :align: center
 
+.. _ultralytics-open-vocabulary-object-detection:
```

Comment:
moved this to the end of the section because the previous sections all use the same dataset defined in the `Setup` section